src/views/mainLayout.js

```diff
@@ -57,7 +57,7 @@ const mainLayout = data => `
           debug_mode: ${Boolean(AppConstants.GA4_DEBUG_MODE)}
         });
       } else {
-        function gtag() {q
```

Comment:
This typo apparently sneaked in here, this else branch is only taken if DNT is disabled, I'll test that as well... we don't have a great way to test this in an automated way (yet, we might be able to get playwright to notice this)